ui/restmodel/src/main/scala/pl/touk/nussknacker/restmodel/component/package.scala

```diff
@@ -16,4 +20,22 @@ package object component {
   @JsonCodec
   final case class ComponentListElement(id: ComponentId, name: String, icon: String, componentType: ComponentType, componentGroupName: ComponentGroupName, categories: List[String], actions: List[ComponentAction], usageCount: Long)
 
+  object ComponentProcess {
+    def apply(nodeId: String, process: BaseProcessDetails[_]): ComponentProcess = ComponentProcess(
+      id = process.processId,
+      name = process.idWithName.name,
+      nodeId = nodeId,
+      isArchived = process.isArchived,
```

Comment:
Can't we return a complex object (scenario with a list of nodes) and let FE decide how to present it?

Reply:
Done